### DISCUSSION TITLE
Translation Systems for Global Knowledge Justice

### DISCUSSION THREAD

**Discussion by mrchristian** [singlesourcepub/community#26] [Events]
Sistemas de traducción para un conocimiento justo y global. Simon Worthington (FORCE11) (English) - A Workshop

An invitation to participate in a workshop for share ideas on translation workflows and ideas in Open Science publishing.

8-11 November 2021, 14:00-15:00 UTC (11:00-12:00 Santiago; 9:00-10:00 Toronto)

Taking place as part of the following event Open and Inclusive Access to Research (OIAR). Organised between partners in South and North America

The objective of the workshop is to create an outline of key issues in translation systems to aid Open Science communities to better coordinate efforts and put in place connected infrastructures. After the workshop GenR wil produce a short guide to the issues involved as part of its Open Science Guides programme. https://genr.eu/wp/guide-needed/

Example systems and groups are:

- CEVOpen using Wikidata https://github.com/petermr/CEVOpen/wiki
- Translate Science - https://translatescience.org/
- Weblate https://weblate.org/

Contact: Simon Worthington simon.worthington@tib.eu

The workshop is a cooperation between FORCE11 and partners: GenR (TIB), CEVOpen https://github.com/petermr/CEVOpen/wiki

The overall event is organised by http://openandinclusiveresearch.org/aboutOIAR/ and sponsored by the following http://openandinclusiveresearch.org/sponsors/

Any questions please drop a line here in the forum

**Comment by xldrkp** [comment]
@mrchristian Great initiative! I could show a workflow that I designed five years ago for translating OER with [Crowdin](http://crowdin.com/). It still works and is still free of charge for Open Source projects.

**Comment by mrchristian** [comment]
Yes please. I'll post a link later today to a list of groups who will give short presentations or inputs to the workshop. And in context of Git and translation at @OSLHannover we have the Open Science Training Handbook which has been translated into 11 languages, very organically but needs some connection and coordination to check on their completeness and allow for updates - see https://github.com/Open-Science-Training-Handbook

**Comment by mrchristian** [comment]
I have written up a fuller description of the workshop.

@xldrkp would be great to get a less than 10 minutes presentation, demo of the CrowdIn workflow, its just what is needed.

https://github.com/Gen-R/organisation/blob/main/translation-systems-guide.md

The time and dates are: 8-11 November 2021, online, 14:00-15:00 UTC (11:00-12:00 Santiago; 9:00-10:00 Toronto)

Register here: http://openandinclusiveresearch.org/register/

Other participants will be>

CEVOpen using Wikidata - https://github.com/petermr/CEVOpen/wiki

CEVOpen is an open research project to build multilingual semantic Atlas of Volatile Phytochemistry as well as a tool kit for others to use. The project uses Text and Data Mining on Open Access literature enable open search and semantification of the scientific literature. CEVOpen is an India, UK, Brazil, Canada founded collaboration.

Translate Science - https://translatescience.org/

We are a new working group that wants to exchange information, lobby and build tools to make translations of scientific articles/reports/books, abstracts, titles and terms more accessible and (thus) stimulate the production of such translations.

Translation Science is currently working on new software for a Translation Switchboard, see the recent blogpost (Nov 2021) https://blog.translatescience.org/building-a-tool-to-find-translated-scientific-articles/

AfricArXiv and Masakhane: Decolonise Science – https://github.com/AfricArxiv/decolonise-science

Recently, as a pilot, one hundred and eighty articles from AfricaArxiv have been translated from French or English into six diverse African languages: isiZulu, Northern Sotho, Yoruba, Hausa, Luganda, and Amharic.

And please all come along, its only an hour long. Hopefully I'll record it too.

Simon

**Reply by xldrkp** [reply]
Thanks @mrchristian! Can I choose a day for the presentation? I have already registered.

**Reply by mrchristian** [reply]
sorry, I realise now fter posting that the dates are misleading, they are the dates of the whole event. This specific workshop is only on the 8th Nov 14:00-15:00 UTC - hope it still works for you

**Reply by mrchristian** [reply]
he trundles off and corrects text :-)

**Reply by mrchristian** [reply]
Thank you for coming along and presenting @xldrkp do you have a link to the Hop-on Study-Buch translation you were talking about? I see the Repo, but not sure how to get to the book view https://collaborating.tuhh.de/hoou-an-der-tuhh-projekte/hop-on-collection/hopon-study-gitbook

**Reply by xldrkp** [reply]
Thanks for having me @mrchristian! It was a pleasure to meet so many folks from around the world working on that topic. Unfortunately we had to take down the website with the HTML part of the book as the content had to be updated and with a lack of financial and human resources that was not possible. But At https://www.hoou.de/materials/hop-on-buch-und-leitfaden-zur-beruflichen-bildung-in-deutschland you will find the archived PDF files of our book based on the formerly free and open source software GitBook. Please get in touch if you want to talk.

**Reply by mrchristian** [reply]
thank you for your contribution - the links here are really helpful. I was asking about the Crowdin workflow as a contributor to The Turing Way is talking about improving tranlation on their workflow. See: https://twitter.com/alejo_coca/status/1459551316951416832?s=20 - I'll point them to your example.

**Comment by xldrkp** [comment]
@mrchristian There was something going on on Twitter related to translation systems. What about having a workshop with the folks doing it to exchange ideas and experience?

**Reply by mrchristian** [reply]
This is a really good idea and your right turns out a number of people are working on this topic at the same time. For GenR I plan to do a short guide for a workflow of recording a book and book series in Wikidata, and for TIB we have the FOSTER Open Science Handbook which needs its existing translations organising, it has 11 langs but not properly coordinated. And I have a related question, but this can be for later about how to document workflows - with a machine readable description and narrative - story telling for the purpose of making these visible so that others can see if processes match theirs. So when do we do the workshop!?

**Reply by mrchristian** [reply]
I'll join this translation workshop today - https://libereurope.eu/mec-events/achieving-digital-language-equality-2030-implications-for-libraries-collections-and-library-users/

**Reply by xldrkp** [reply]
> how to document workflows - with a machine readable description and narrative - story telling

That sounds interesting! Any tools in mind to do so?

**Reply by mrchristian** [reply]
No, that's why I'm mentioning it. It was an idea I had after a workshop yesterday as it seems nearly everyone is failed by the current provision of publishing tools from - the market, government, institutions, event the community - that telling the stories about workflows and the benefits that could be had if things worked better would be a good idea. So we need narrative and story telling. But I though there must be some LOD way of describing a workflow and tools versions used, problems, successes. Like a Software Citation / Workflow LOD / storytelling - slasher fiction | ASAPbio did it a bit for Open Peer Review https://reimaginereview.asapbio.org/

**Reply by xldrkp** [reply]
So, let's talk. Could be interesting for others, too. What about an open invitation for a discussion that would be organized by our community and be announced in Events here?

**Reply by mrchristian** [reply]
Open invite - yep OK. Lets put out the call, Im in the LIBER workshop on tranlation - I'll invite people in and we can see what to do.

**Reply by xldrkp** [reply]
Great! Let me know where I can help.

**Reply by VictorVenema** [reply]
Sounds interesting. Do let me know. We can also spread the announcement on the [translate.science email distribution list](https://lists.riseup.net/www/info/translate.science); I expect more would be interested.

**Comment by xldrkp** [comment]
@mrchristian Yesterday I shared the idea of story telling for documentation with the Fab City community in Hamburg. Comes out they are looking for tooling, too. For whatever we plan we could invite them.
